core/windows/winmain.cpp

```diff
@@ -810,3 +812,9 @@ void os_DoEvents()
 
 int get_mic_data(u8* buffer) { return 0; }
 int push_vmu_screen(u8* buffer) { return 0; }
+
+void os_LaunchFromURL(const string& url)
+{
+	auto cmd = "start " + url; 
+	system(cmd.c_str());
```

Comment:
Ew. There's windows APIs for that, which don't involve the "start" program :P
e.g. 
```c
ShellExecuteA(0, 0, url.c_str(), 0, 0 , SW_SHOW );
```

Reply:
Mnn, can you implement it this way and append on the PR? I don't have a windows env setup, so this was 100% guesswork

Reply:
Done.

Reply:
Thx. Does it work as expected on windows?

Reply:
Yup! Executing the "open" verb of an URL will open it in the default browser, same as if you were opening a document.